src/errors.ts

```diff
@@ -33,8 +33,19 @@ class WebhookAuthenticateError extends Error {
   }
 }
 
+class AttachmentDecryptionError extends Error {
+  constructor(message = 'Attachment decryption with the given nonce failed.') {
+    super(message)
+    this.name = this.constructor.name
+    // Set the prototype explicitly.
+    // See https://github.com/facebook/jest/issues/8279
+    Object.setPrototypeOf(this, AttachmentDecryptionError.prototype)
```

Comment:
I don't think you need these because this error is never tested for in Jest (and can't be tested since it's meant to be caught within the same function)